test/python/test_circuit.py

```diff
@@ -48,8 +48,8 @@ def test_circuit_qasm(self):
         """
         qr1 = QuantumRegister(1, 'qr1')
         qr2 = QuantumRegister(2, 'qr2')
-        c = ClassicalRegister(3, 'c')
-        qc = QuantumCircuit(qr1, qr2, c)
+        cr = ClassicalRegister(3, 'cr')
```

Comment:
There is a test failing, something related to changing the internal names of registers and then comparing with some "expected qasm". You can preserve the internal name of the register or fix the test accordingly.